python_tests/test_python_frontend.py

```diff
@@ -60,8 +60,14 @@ def inner(*args, **kwargs):
         # if ("new_fusion_expected" not in kwargs) or kwargs["new_fusion_expected"]:
         #    FusionCache.reset()
 
+        # skip_serde_check is only used by the decorator so remove it before running test_fn
+        skip_serde_check = kwargs.pop("skip_serde_check", False)
+
         # Run test to populate FusionCache
-        test_fn(*args, **kwargs)
+        result = test_fn(*args, **kwargs)
```

Comment:
I think in the perfect world we would like to verify test result before/after serde, excluding random generatos.

Are we checking that after deserialization, we are indeed calling the deserialized entries in the cache rather than just compiling and generating new things?

Reply:
nvm. I see that we have overwritten `new_fusion_expected`